standard/expressions.md

```diff
@@ -888,22 +897,25 @@ The ***inferred return type*** is determined as follows:
 >
 > and the result is of type `IEnumerable<string>`.
 >
-> The following example demonstrates how anonymous function type inference allows type information to “flow” between arguments in a generic method invocation. Given the method:
+> The following example demonstrates how anonymous function type inference allows type information to “flow” between arguments in a generic method invocation. Given the following method and invocation:
 >
 > ```csharp
-> static Z F<X,Y,Z>(X value, Func<X,Y> f1, Func<Y,Z> f2)
+> using System;
+>
+> class A
```

Comment:
Again, I suspect we could have a library template for "there's already a class declaration, put the given code inside it".

Reply:
I left it as I proposed. The example is still marked "incomplete," so we can revisit it later.